tests/strands/multiagent/test_base.py

```diff
@@ -177,7 +177,7 @@ async def invoke_async(self, task, invocation_state, **kwargs):
         def serialize_state(self) -> dict:
             return {}
 
-        def deserialize_state(self, payload: dict) -> None:
+        def restore_from_session(self, payload: dict) -> None:
```

Comment:
Why is `serialize_state` different from `restore_from_session`?  Additionally - 
We just introduced these methods, so why are we now renaming these? 

If we're changing one we need to change the other and before we make the next release, we need to decide what the namings are - once these ship we won't be able to change them.